Update for .NET Core RTM

tillig opened the pull request:
This is for #111.

I had to comment out one of the unit tests about setting defaults (marked with a TODO) because it didn't validate.

I also sorted/tidied the project.json files for easier diff comparisons.

tillig commented:
Oh, one thing - I had to switch the unit tests to only test one framework rather than multiple frameworks. There's a bug in VS tooling and xunit such that VS won't let you run tests that target multiple frameworks. If you can figure that one out... Well, I spent a couple hours on it and have up. :)

domaindrivendev commented on src/Swashbuckle.Swagger/project.json:
```diff
@@ -1,27 +1,28 @@
 ﻿{
```
I kinda liked the existing layout for the project.json files :( I'd prefer to have properties grouped by meaning as opposed to alphabetically. Any chance we could change these back?

tillig replied:
I can definitely do that. I changed them because in a solution with a lot of projects that all have to change at the same time it's much harder to diff and compare what's missing when they're in that grouped format. Having them in a predictable format, while potentially less human-readable, fixes a lot of problems with respect to long-term maintainability.

But, sure, I'll change them back. It'll take a bit.

domaindrivendev commented on src/Swashbuckle.Swagger/project.json:
```diff
@@ -1,27 +1,28 @@
+      "Swagger Documentation Discovery Help WebApi AspNet AspNetCore"
+    ]
+  },
+  "version": "6.0.0-rtm1"
```
Version should be 6.0.0-beta10. Not ready to call this baby rtm or even close just yet

tillig replied:
Unfortunately it can't be `beta10`. The problem is that NuGet uses the suffix _alphabetically_ to determine what's new. So, alphabetically, `beta10` is _older_ than `beta9`. That's actually why they switched early on from releasing `-rc2` style suffixes to `-rc2-final` - because people using early drops couldn't _upgrade_ to an _older_ version (`-rc2` < `-rc2-12345` < `-rc2-final`). When I have CI hooked up, I usually use the build number (~3 digits) as the beta suffix so I don't run into this issue.

I arbitrarily picked `rtm` so when I published to our internal NuGet server it would be seen as newer. For now I'll switch to `-rc-001` so it'll work with NuGet but won't be seen as RTM ready.

domaindrivendev commented on test/WebSites/Basic/Swagger/AddCartDefault.cs:
```diff
@@ -17,9 +18,9 @@ public void Apply(Schema model, ModelFilterContext context)
+            model.Default = new Cart
             {
-                Id = "myCartId"
+                Id = 123
```
Right now - anonymous types are better for setting examples because the Swagger-specific serializer may not reflect the API's serialization settings (camelCasing etc.). At least with the anonymous types, you can control the casing yourself.

domaindrivendev commented on test/WebSites/CustomizedUi/project.json:
```diff
@@ -1,81 +1,52 @@
-    "Swashbuckle.SwaggerGen": "6.0.0-beta9",
-    "Swashbuckle.SwaggerUi": "6.0.0-beta9"
+    "Swashbuckle.SwaggerGen": "6.0.0-rtm1",
+    "Swashbuckle.SwaggerUi": "6.0.0-rtm1"
```
6.0.0-beta10

tillig commented:
I reformatted the project.json files to hopefully be closer to the originals. I did notice from the ASP.NET Core team's projects that their tags are all lower case and they're not space-delimited anymore, so I updated that, too.

I went with `-rc-001` for the version to fix the `beta` vs `rtm` issue. NuGet should see the `rc` version as later than `beta` yet it still indicates not quite ready for prime time.

Finally, I added an `.editorconfig` to help keep your style settings in place. If you don't use the [EditorConfig](http://editorconfig.org/) add-ins it won't matter, but if you do it'll help. I work across a lot of projects that have different indentation requirements (tabs vs. spaces, four spaces vs. two spaces, etc.) and if you use the plugin it'll automatically adjust on a per-project basis for you.

domaindrivendev commented:
@tillig - thanks for this great contribution. Great catch on the version ordering. However, the current state of this project is best reflected through a "beta" version. There's still quite a few missing features and there's also one or two areas where I foresee the need for breaking changes. It's a little ugly, but I ended up going with the following - 6.0.0-beta901

domaindrivendev commented:
Nuget shipping shortly ...

tillig commented:
Sounds great! Glad I could help.

another-guy commented:
Guys, don't want to make noise in this PR, but be sure your efforts are appreciated!
